### DISCUSSION TITLE
Show the hours in rows in the Calendar

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/PublishPress-Planner#1294] [Feature Requests]
One big publisher says this:

"The Calendar does not display time - we publish many articles a day and being able to see what time each is scheduled for, rather than having to use a metadata field that is a pain to change every time we move content instead of being fed automatically from the time set in a post, would be very useful.

In the same vein, when we move an item on the calendar, we would like the time to change as well as the date (maybe it would ask us the new time after we drop it on a new day?). This is not as critical."

In short, the goal is a more Google-Calendar like experience:
![Screen Shot 2019-10-30 at 2 12 47 PM](https://user-images.githubusercontent.com/3868028/67887546-d8b8e800-fb21-11e9-9fc4-707888992fa4.png)

**Comment by stevejburge** [comment]
Related:
https://github.com/publishpress/PublishPress/issues/457
https://github.com/publishpress/PublishPress/issues/460
